docs/dex-blog-post.md

```diff
@@ -55,93 +55,20 @@ protocol). Additional registers should also be specified as required by
-        }
-      ],
-      "additionalRegisters": {
-        "R4": "0e03544b4e",
-        "R5": "0e0d47656e6572696320746f6b656e",
-        "R6": "0e0132"
-      },
-      "transactionId": "5d899e1551b324012c6c17636422fceb8077c444585de06f444fe680f9badd92",
-      "index": 0
-    },
-    {
-      "boxId": "d35162679ff1479df4851020278ad83e6fdc78acfde1e66fe1efffc9869ab41a",
-      "value": 1000000,
-      "ergoTree": "ErgoTree(16,WrappedArray(IntConstant(0), IntConstant(0), ConstantNode(Coll(16,2,4,-96,11,8,-51,2,121,-66,102,126,-7,-36,-69,-84,85,-96,98,-107,-50,-121,11,7,2,-101,-4,-37,45,-50,40,-39,89,-14,-127,91,22,-8,23,-104,-22,2,-47,-110,-93,-102,-116,-57,-89,1,115,0,115,1),Coll[SByte$]), ConstantNode(Coll(1),Coll[SInt$]), IntConstant(1)),Right(BoolToSigmaProp(AND(ConcreteCollection(WrappedArray(EQ(Height$(163),SelectField(ExtractCreationInfo(ByIndex(Outputs$(165),ConstantPlaceholder(0,SInt$),None)),1)), EQ(ExtractScriptBytes(ByIndex(Outputs$(165),ConstantPlaceholder(1,SInt$),None)),SubstConstants(ConstantPlaceholder(2,Coll[SByte$]),ConstantPlaceholder(3,Coll[SInt$]),ConcreteCollection(WrappedArray(CreateProveDlog(DecodePoint(MinerPubkey$(172)))),SSigmaProp))), EQ(SizeOf(Outputs$(165)),ConstantPlaceholder(4,SInt$))),SBoolean)))),4836,[B@713ec32d)",
-      "creationHeight": 123414,
-      "assets": [],
-      "additionalRegisters": {},
-      "transactionId": "5d899e1551b324012c6c17636422fceb8077c444585de06f444fe680f9badd92",
-      "index": 1
-    },
-    {
-      "boxId": "57dc06e5a197e15d9a73804735231d6c240ca6bdd45753073883c79cf726ea96",
-      "value": 9951000000,
-      "ergoTree": "ErgoTree(0,WrappedArray(),Right(ConstantNode(SigmaProp(ProveDlog(ECPoint(6ba5cf,8ae5ac,...))),SSigmaProp)),80,[B@34d713a2)",
-      "creationHeight": 123414,
-      "assets": [],
-      "additionalRegisters": {},
-      "transactionId": "5d899e1551b324012c6c17636422fceb8077c444585de06f444fe680f9badd92",
-      "index": 2
-    }
-  ]
-}
-Sending the transaction... Ok
-Server returned tx id: 5d899e1551b324012c6c17636422fceb8077c444585de06f444fe680f9badd92
-``` 
+Here is an example of using `dex:IssueToken` -
+https://gist.github.com/greenhat/47fc2a4190ba8874248a3f759f645f1b
 
 ## Sell Tokens
 
-If we have tokens in a box we can sell them. Well, at least we can create an Ask order
-and submit it in the order book. In our DEX implementation we create orders and store
-them directly on the Ergo blockchain. The created order in our case is a box (seller box)
-protected with the special _seller contract_ holding the necessary amount of ERGs (`ergAmount`). 
+If we have tokens in a box we can sell them. Well, at least we can create an Ask order and submit it
+to the order book. In our DEX implementation we create orders and store them directly on the Ergo
+blockchain. The created order in our case is a box (seller box) protected with the special _seller
+contract_ holding the minimum amount of ERGs (`minErg`) and `tokenAmount` of the `TID` tokens.
```

Comment:
@aslesarenko Mentioning `minErg`, `tokenAmount` and `TID` might confuse the reader, that expecting to see them in the contract below.

Reply:
We need to have a separate section with overall diagram and its explaination.